windows-image-install.ps1

```diff
@@ -0,0 +1,50 @@
+# Following script works with the Windows 2016 image provided by GCE. If running a different version of Windows/Powershell, changes may be needed
+# We are in the first phase where we need to configure PowerShell, install Chocolately, install the OpenSSH Server.
+Write-Output "Setting execution policy for PowerShell scripts...";
+Set-ExecutionPolicy Bypass -Scope Process -Force;
+
+Write-Output "Installing Chocolatey...";
+Invoke-Expression ((New-Object System.Net.WebClient).DownloadString('https://chocolatey.org/install.ps1'));
+
+Write-Output "Refreshing environment...";
+RefreshEnv.cmd
+
+Write-Output "Installing OpenSSH Server..."
+choco install -y openssh -params '"/SSHServerFeature /KeyBasedAuthenticationFeature"'
+
+if ($LastExitCode -ne 0) {
+exit 1
+}
+
+Write-Output "Installing Java 8..."
+choco install -y jre8
+
+# No need to add a user if you've already configured one
```

Comment:
The use of punctuation is inconsistent throughout this file's comments. Suggest adding the missing '.' throughout.

Reply:
done